src/apps/core/views/monster-language-selector.html

```diff
@@ -0,0 +1,7 @@
+<select{{#each attributes}} {{@key}}="{{this}}"{{/each}}>
+{{#select selectedLanguage}}
+	{{#each languages}}
+		<option value="{{this}}">{{tryI18n @root.i18n.monsterLanguages this}}</option>
```

Comment:
This list should be alphabetically sorted based on the i18n labels. In this case, you'll have to resolve the i18n references outside of the template:
```javascript
var languages = _
	.chain(monster.supportedLanguages)
	.clone()
	.map(function(code) {
		value: code,
		label: monster.util.tryI18n(monster.apps.core.i18n.active().monsterLanguages, code)
	})
	.value();
```

Don't forget to take care of the default option conditionally added as well.